### DISCUSSION TITLE
Strip HTML tags when indexing

### DISCUSSION THREAD

**Discussion by marcuslindblom** [ravendb/ravendb#17634] [Q&A] [answered]
Do you have any recommendation on how to remove all HTML tags when indexing? Maybe you have som helper function or do I just write my own?

**Comment by ppekrol** [comment, marked as answer]
I think it would be best to create your own indexing extension method via additional assemblies and take leverage of e.g. https://www.nuget.org/packages/HtmlAgilityPack
Documentation: https://ravendb.net/docs/article-page/6.0/csharp/indexes/additional-assemblies

In 3.5 there was a extension method that used that library with following code:
```
        public static dynamic StripHtml(dynamic o)
        {
            string value = o as string;

            if (value  == null)
                return DynamicNullObject.Null;

            if (value == string.Empty)
                return value;

            var document = new HtmlDocument();
            document.LoadHtml(value);

            return document.DocumentNode.InnerText.Trim();
        }
```
